CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs need to be appropriately assigned and labelled.
```

Comment:
For community contributors, there is no way to label and request review for the pull request, so it must be done by someone with access to the main repository.

Reply:
Updated the instructions to clarify this.